base/docs/utils.jl

```diff
@@ -270,12 +282,17 @@ const builtins = ["abstract", "baremodule", "begin", "bitstype", "break",
 
 moduleusings(mod) = ccall(:jl_module_usings, Any, (Any,), mod)
 
-filtervalid(names) = filter(x->!ismatch(r"#", x), map(string, names))
+is_valid_name(name::AbstractString) = !ismatch(r"#", name)
 
-accessible(mod::Module) =
-    [names(mod, true, true);
-     map(names, moduleusings(mod))...;
-     builtins] |> unique |> filtervalid
+function accessible(mod::Module)
+    result = ByteString[]
+    append!(result, filter(is_valid_name, map(string, names(mod, true, true))))
+    for inner_mod in moduleusings(mod), sym in names(inner_mod)
+        name = string(sym)
+        if is_valid_name(name); push!(result, name) end
```

Comment:
Small style for similar one-liners is

    is_valid_name(name) && push!(result, name)

But not very important, LGTM.